cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/batch/source/DBSource.java

```diff
@@ -60,6 +64,11 @@
     "import from the specified table. Examples: SELECT COUNT(*) from <my_table> where <my_column> 1, " +
     "SELECT COUNT(my_column) from my_table. NOTE: Please include the same WHERE clauses in this query as the ones " +
     "used in the import query to reflect an accurate number of records to import.";
+  private static final String COLUMN_CASE_DESCRIPTION = "Sets the case of the column names returned from the query." +
+    "Possible options are UPPER, lower. By default or for any other input, the column names are not modified" +
```

Comment:
no, I just meant in the description does it throw you off (that these are the only two values supported). I saw that the code accepts any case, which is good.